CMakeLists.txt

```diff
@@ -40,6 +40,32 @@ if (NOT CMAKE_BUILD_TYPE)
   set (CMAKE_BUILD_TYPE Debug)
 endif (NOT CMAKE_BUILD_TYPE)
 
+## Retrieve git revision (at configure time)
+find_program (GIT_EXECUTABLE git DOC "git command line client")
```

Comment:
Would be better to use FindGit instead (https://gitlab.kitware.com/cmake/cmake/blob/master/Modules/FindGit.cmake)